SPECS/mesa-libGLU/mesa-libGLU.spec

```
@@ -1,18 +1,19 @@
 Summary:        Mesa libGLU library
 Name:           mesa-libGLU
 Version:        9.0.1
-Release:        3%{?dist}
+Release:        4%{?dist}
 License:        MIT
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 URL:            https://mesa3d.org/
-Source0:        https://ftp.freedesktop.org/pub/mesa/glu/glu-%{version}.tar.xz
-Source1:        %{name}-LICENSE.txt
+Source0:        https://mesa.freedesktop.org/archive/glu/glu-%{version}.tar.xz
+Source1:        LICENSE.PTR
+Source2:        make-git-snapshot.sh
 BuildRequires:  autoconf
 BuildRequires:  automake
-BuildRequires:  gcc-c++
 BuildRequires:  libtool
 BuildRequires:  mesa-libGL-devel
+#Requires:
 Provides:       libGLU = %{version}-%{release}
```

Nit: accidentally added comment.

```suggestion
Provides:       libGLU = %{version}-%{release}
```

---

Will do on next round clean by separate PR.